tests/bud/volume-ownership/Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM alpine
-ADD test-ownership /usr/bin/
 RUN adduser -D -H testuser && addgroup testgroup
 RUN mkdir -p /vol/subvol
 RUN chown testuser:testgroup /vol/subvol
```

Comment:
Out of curiosity, what is the purpose of the `touch /test` two lines down? (not an addition in this PR; you'll need to click the expando). I'm assuming it's a copy-paste artifact, and unnecessary for purposes of this test. If so, as long as you're making changes here, could you perhaps remove it to prevent future maintainers from spending time wondering about it? (Or if it's real and necessary, perhaps document why?) Thanks!

Reply:
Actually it is necessary. The [original bug](https://github.com/containers/buildah/issues/2202) only showed up when the volume cache was in play. And that only happens if there are additional `RUN` commands after the `VOLUME` command.

I chose that particular command because it obviously shouldn't have changed the permissions of the volume ... but it did.

I can add a commit with a comment in the Dockerfile to try to summarize the reasoning if you think it would be helpful.

Reply:
Given what you just said, I think a comment is not only helpful but critical: a future maintainer might have no idea of this context, and could easily refactor/optimize away the `touch`. It is imperative to explain the why, especially in non-obvious cases such as this one. Thanks for your quick explanation.

Reply:
Added a comment. Not sure what your feelings are about github links in source, but the comment above the `Preserve` function has a bunch of information about this.